docs/fsharp/language-reference/xml-documentation.md

```diff
@@ -3,54 +3,149 @@ title: XML Documentation
+There are two ways to write XML documentation comments. One is to just write the documentation directly in a triple-slash
+comment, without using XML tags. If you do this, the entire comment text is taken as the summary documentation for the code
+construct that immediately follows. Use this method when you want to write only a brief summary for each construct. The other
+method is to use XML tags to provide more structured documentation. The second method enables you to specify separate notes
+for a short summary, additional remarks, documentation for each parameter and type parameter and exceptions thrown, and a description of the return value.
 
-For more information about how to generate documentation from XML, see [XML Documentation Comments &#40;C&#35; Programming Guide&#41;](../../csharp/programming-guide/xmldoc/index.md).
+## Comments without XML tags
+
+If a comment does not start with a `<` then the comment is interpreted as a `<summary>` section. The comment is encoded to XML during documentation preparation,
+so characters such as `<`, `>` and `&` need not be escaped. If you don't specify a summary tag explicitly, you should
+not specify other tags, such as **param** or **returns** tags.
+
+The following example shows the alternative method, without XML tags. In this example, the entire text in the comment is considered a summary.
+
+[!code-fsharp[Main](~/samples/snippets/fsharp/lang-ref-2/snippet7102.fs)]
+
+## Comments with XML tags
+
+If the comment body begins with `<` then it is treated as an XML formatted comment body using XML tags. The following is a typical XML documentation comment in a signature file:
+
+[!code-fsharp[Main](~/samples/snippets/fsharp/lang-ref-2/snippet7101.fs)]
 
 ## Recommended Tags
 
-There are two ways to write XML documentation comments. One is to just write the documentation directly in a triple-slash comment, without using XML tags. If you do this, the entire comment text is taken as the summary documentation for the code construct that immediately follows. Use this method when you want to write only a brief summary for each construct. The other method is to use XML tags to provide more structured documentation. The second method enables you to specify separate notes for a short summary, additional remarks, documentation for each parameter and type parameter and exceptions thrown, and a description of the return value. The following table describes XML tags that are recognized in F# XML code comments.
+If using XML tags, the following table describes the outer tags recognized in F# XML code comments.
 
 |Tag syntax|Description|
```

Comment:
I would code-fence the Tag syntax for all tables in the article to avoid over-localization.

Reply:
ok

Reply:
Done